proposal.html

```diff
@@ -485,7 +485,7 @@
 
   <ins class="block">
     <emu-alg>
-      1. If _func_ is a <emu-xref href="#sec-bound-function-exotic-objects">Bound Function exotic object</emu-xref> or a <emu-xref href="#sec-ecmascript-standard-built-in-objects">built-in Function object</emu-xref>, then return an implementation-dependent String source code representation of _func_. The representation must have the syntax of a |NativeFunction|.
+      1. If _func_ is a <emu-xref href="#sec-bound-function-exotic-objects">Bound Function exotic object</emu-xref> or a <emu-xref href="#sec-ecmascript-standard-built-in-objects">built-in Function object</emu-xref>, then return an implementation-dependent String source code representation of _func_. The representation must have the syntax of a |NativeFunction|. Additionally, if _func_ is a <emu-xref href="#sec-well-known-intrinsic-objects">Well-known Intrinsic Object</emu-xref> and the Global Name associated with _func_ has the syntax of a |BindingIdentifier|, the portion of the returned String that would be matched by |BindingIdentifier| must be the same as the Global Name associated with _func_.
```

Comment:
No, I get that, but I also get that
> 1. If A, then X.
2. Else if B, then Y.
3. Else if C, then X.

is equivalent to
> 1. If B and not A, then Y.
2. Else if A or C, then X.

But it is _not_ clear to me that the new "Additionally" clause can be omitted from step 3 (hence the initial comment).